.github/workflows/ci.yml

```diff
@@ -102,13 +102,13 @@ jobs:
       - name: cargo check
         run: cargo check --workspace --all-features --lib --bins
 
-  minimal-crates:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: dtolnay/rust-toolchain@nightly
-      - uses: swatinem/rust-cache@v2
-      - name: cargo check
-        run: |
-          rm -f Cargo.lock
-          cargo +nightly check -Z minimal-versions --workspace --all-features --lib --bins
+  # minimal-crates:
```

Comment:
I know this sucks, but lets bring this back before merging please

Reply:
I just plain don't know how to fix it. I am getting an error in `ahash`, which is not even a dependency

```
cargo +nightly tree --all-features  | grep ahash
<crickets>
```

Reply:
I'll check

Reply:
Pushed an update that changes CI to this:

```
cargo +nightly check -Z minimal-versions -p quic-rpc -p quic-rpc-derive --all-features --lib --bins
```

Error is gone, but I am not super confident that it is doing anything at all anymore...